File: PYTHON/Cap05/c05ex04.py
# c05ex04.py

resp = "S"
while not (resp.upper() != "S"):
  n = int(input("Entre um valor numerico: "))

  r = n * 3
  print("Resultado =", r)
  print()

  resp = input("Deseja continuar? (S/N): ").upper()[0]

  print()

input("\nTecle <Enter> para encerrar... ")

File: DART/Cap05/c05ex04.dart
// c05ex04.dart

import 'dart:io';

void main() {
  String RESP = 'S';
  int N;
  int R;

  while (!(RESP != 'S')) {
    stdout.write("Entre um valor numerico: ");
    N = int.tryParse(stdin.readLineSync() ?? '') ?? 0;

    R = N * 3;

    print("Resultado = $R\n");

    stdout.write("Deseja continuar? (S/N): ");
    RESP = stdin.readLineSync()?.toUpperCase() ?? 'N';

    print("");
  }

  stdout.write("\nTecle <Enter> para encerrar... ");
  stdin.readLineSync();
}
